File: python/demo/chart1/Student.py
import People as p;
class Student(p.People):
	grade = 1;
	
	def __init__(self, name, age, grade):
		p.People.__init__(self,name, age);
		self.grade = grade;
		
	def speak(self):
		print("%s 说: 我 %d 岁了. 我读 %d 级了!"%(self.name, self.age, self.grade));

File: java/aopsample/src/main/java/com/bobo/aopsample/bean/Student.java
package com.bobo.aopsample.bean;

import java.io.Serializable;

import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.cloud.context.config.annotation.RefreshScope;

@RefreshScope
@ConfigurationProperties(prefix="student")
public class Student implements Serializable{

	/**
	 * 
	 */
	private static final long serialVersionUID = 2913472053130108101L;
	private String id;
	private String name;
	private Integer age;
	public String getId() {
		return id;
	}
	public void setId(String id) {
		this.id = id;
	}
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	public Integer getAge() {
		return age;
	}
	public void setAge(Integer age) {
		this.age = age;
	}
}
